lib/access_check_filter.js

```diff
@@ -64,6 +66,9 @@ module.exports = (hyper, req, next, options, specInfo) => {
                 }
 
                 return contentPromise.then((content) => redirect(content, location, options));
+            } else if (content.status === 302) {
```

Comment:
When we're taking the content from the storage we don't set 302 and rely on the filter to get it from the restrictions table